AvroSourceGenerator/AvroGenerator.cs

```diff
@@ -33,13 +33,13 @@ public void Initialize(IncrementalGeneratorInitializationContext initContext)
             .Where(file => _avroSchemasToGenerate.Contains(Path.GetFileNameWithoutExtension(file.Path)));
 
         var schemaFileNameAndContents = schemaFiles.Select(
-            static (file, cancellationToken) =>
-                (name: Path.GetFileNameWithoutExtension(file.Path),
-                    content: file.GetText(cancellationToken)!.ToString()));
+                static (file, cancellationToken) =>
+                    (name: Path.GetFileNameWithoutExtension(file.Path),
+                        content: file.GetText(cancellationToken)!.ToString()))
+            .Collect();
 
         initContext.RegisterSourceOutput(schemaFileNameAndContents,
-            static (spc, nameAndContent) =>
-                spc.AddSource($"AvroGeneratedSchemas.{nameAndContent.name}",
-                    AvroUtils.GenerateSourceCode(nameAndContent.content)));
+            static (spc, namesAndContents) =>
+                spc.AddSource("AvroGeneratedSchemas", AvroUtils.GenerateSourceCode(namesAndContents)));
```

Comment:
I don't _love_ glomming everything into a single file, feels like the kind of thing where a single file size limit might get hit one day. Needs more investigation.